CCPAPI/sip/CMakeLists.txt

```diff
@@ -0,0 +1,149 @@
+##########################################################################
+#                                                                        #
+#                               SIP                                      #
+#                                                                        #
+#  This program is free software; you can redistribute it and/or modify  #
+#  it under the terms of the GNU Library General Public License as       #
+#  published by the Free Software Foundation; version 2 or later of the  #
+#  License.                                                              #
+#                                                                        #
+#  This program is distributed in the hope that it will be useful,       #
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+#  GNU General Public License for more details.                          #
+#                                                                        #
+#          Copyright 2020 Paul RASCLE www.openfields.fr                  #
+#                                                                        #
+##########################################################################
+
+cmake_minimum_required(VERSION 3.0)
+cmake_policy(SET CMP0071 NEW)
+message(STATUS "SIP processing")
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake" )
```

Comment:
Adding to `CMAKE_MODULE_PATH` probably belongs in the top-level file, not in each cmake file.